src/upload-tests.proj

```diff
@@ -121,8 +125,37 @@
   <!-- main Entrypoint -->
   <Target Name="Build">
     <Message Text="CoreFX-specific Helix test upload project, beginning submission to Helix" />
-    <Message Condition="'$(EnableCloudTest)' == 'false'" Text="... skipping upload / submission due to property 'EnableCloudTest' being set to 'false'" />
+    <Message Condition="'$(EnableCloudTest)' == 'false'" Text="... will skip upload / submission due to property 'EnableCloudTest' being set to 'false'" />
+    <Message Condition="'$(SkipSendToHelix)' == 'true'"  Text="... will skip submission due to property 'SkipSendToHelix' being set to 'true'" />
     <CallTarget Condition="'$(EnableCloudTest)' != 'false'" Targets="CoreFXPreCloudBuild;CompressRuntimeDir;HelixCloudBuild" />
+    <CallTarget Condition="'$(LocalHelixExecution)' == 'true'" Targets="ExecuteLocally" />
+
   </Target>
-  
+
+  <!-- Local Execution entrypoint, can be invoked directly -->
+  <!-- Sample direct command line:
+          msbuild upload-tests.proj /t:ExecuteLocally /p:LocalExecutionTestList=C:\GIT\corefx\bin\tests\Windows_NT.AnyCPU.Release\archive\TestList.json 
+                                    /p:LocalExecutionTestFilter=System.IO
+  -->
+  <Target Name="ExecuteLocally">
+    <PropertyGroup>
+      <LocalExecutionTestList Condition="'$(LocalExecutionTestList)'==''">$(HelixLogFolder)$(TestListFilename)</LocalExecutionTestList>
+      <LocalExecutionCorrelationId Condition="'$(LocalExecutionCorrelationId)'==''"></LocalExecutionCorrelationId>
```

Comment:
mostly for folks who are using this project for local execution debugging to be reminded to set the property.  I find it preferable to using an error when I'm running something a bunch.